src/deluge/gui/menu_item/defaults/ui/clip_type/default_new_clip_type.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2014-2023 Synthstrom Audible Limited
+ *
+ * This file is part of The Synthstrom Audible Deluge Firmware.
+ *
+ * The Synthstrom Audible Deluge Firmware is free software: you can redistribute it and/or modify it under the
+ * terms of the GNU General Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program.
+ * If not, see <https://www.gnu.org/licenses/>.
+ */
+#pragma once
+#include "definitions_cxx.hpp"
+#include "gui/l10n/l10n.h"
+#include "gui/menu_item/selection.h"
+#include "storage/flash_storage.h"
+#include "util/misc.h"
+
+namespace deluge::gui::menu_item::defaults::ui::clip_type {
+class DefaultNewClipType final : public Selection {
+public:
+	using Selection::Selection;
+	void readCurrentValue() override { this->setValue(FlashStorage::defaultNewClipType); }
+	void writeCurrentValue() override { FlashStorage::defaultNewClipType = this->getValue<OutputType>(); }
+	deluge::vector<std::string_view> getOptions() override {
+		using enum l10n::String;
+		return {
+		    "SYNTH", "KIT", "MIDI", "CV", "AUDIO",
```

Comment:
this should be using the output type enum, this will break if they're reordered

Reply:
> this should be using the output type enum, this will break if they're reordered

will fix this is in the main branch and cherry pick after

Reply:
don't bother cherry picking, this branch isn't getting future development so it doesn't matter